src/FastRandom.cpp

```diff
@@ -8,17 +8,19 @@
 
 #include <random>
 
-#ifdef _WIN32
-	#define thread_local static __declspec(thread)
-#elif defined __APPLE__
-	#define thread_local static __thread
+#if defined (__GNUC__)
+	#define ATTRIBUTE_TLS static __thread
```

Comment:
As in, you should add a separate definition for normal Linux systems for the standard attribute, in addition to the current Mac + Windows definitions, but nevermind :P